src_ts/components/pages/page-one/page-one-tabs.ts

```diff
@@ -43,8 +43,10 @@ export class PageOneTabs extends connect(store)(LitElement) {
         <h1 slot="page-title">Title here</h1>
 
         <div slot="title-row-actions" class="content-header-actions">
-          <paper-button raised>Action 1</paper-button>
-          <paper-button raised>Action 2</paper-button>
+          <available-actions
+            .entityId="${this.record?.id}"
+            .actions="${this.record?.actions_available || []}"
+          ></available-actions>
```

Comment:
Was expecting another component that inherits from available-actions, to set the stage for apps that have more than one module => action component for module 1, action component for module2. Each inheriting from available-actions